cli/js/lib.deno.ns.d.ts

```diff
@@ -114,19 +83,14 @@ declare namespace Deno {
     skip?: string | RegExp;
     /** Disable logging of the results. Defaults to `false`. */
     disableLog?: boolean;
-    /** Custom reporter class. If not provided uses console reporter. */
-    reporter?: TestReporter;
+    /** If true, report results to the console as is done for `deno test`. Defaults to `false`. */
+    reportToConsole?: boolean;
```

Comment:
What's the reason for defaulting to `false`?

Reply:
It's programmatic API, if people are using it over `deno test` it should probably be without the side-effect by default, right?

Reply:
Reverted.